docs/ovs-pipeline.md

```diff
@@ -376,28 +377,41 @@ In the future this table may support an additional mode of operations, in which
 it will implement kube-proxy functionality and take care of performing
 load-balancing / DNAT on traffic destined to services.
 
-### CnpEgressRuleTable (45)
+### CnpEgressRuleTables (45-49)
 
-For this table, you will need to keep mind the CNP [specification](#antrea-network-policy-crd-implementation)
+For these tables, you will need to keep mind the CNP [specification](#antrea-network-policy-crd-implementation)
 that we are using.
 
-This table is used to implement the egress rules across all ClusterNetworkPolicies.
-If you dump the flows for this table, you should see something like this:
+These tables are used to implement the egress rules across all ClusterNetworkPolicies.
+Depending on the tier to which the CNP belongs to, the rules will be installed
+in a table corresponding to that tier. The egress CNP table to tier mappings is as
+follows:
+```
+1. Emergency tier -> EmergencyEgressRuleTable(45)
+2. SecurityOps tier -> SecurityOpsEgressRuleTable(46)
+3. NetworkOps tier -> NetworkOpsEgressRuleTable(47)
+4. Platform tier -> PlatformEgressRuleTable(48)
+5. Application tier -> ApplicationEgressRuleTable(49)
+```
+Since the example CNP resides in the Application tier, if you dump the flows for
+table 49, you should see something like this:
 ```
-1. table=45, priority=64990,ct_state=-new+est,ip actions=resubmit(,70)
-2. table=45, priority=11800,conj_id=2,ip actions=load:0x2->NXM_NX_REG6[],resubmit(,70)
-3. table=45, priority=11800,ip,nw_src=10.10.1.6 actions=conjunction(2,1/3)
-4. table=45, priority=11800,ip,nw_dst=10.10.1.8 actions=conjunction(2,2/3)
-5. table=45, priority=11800,udp,tp_dst=53 actions=conjunction(2,3/3)
-6. table=45, priority=0 actions=resubmit(,50)
+1. table=49, priority=64990,ct_state=-new+est,ip actions=resubmit(,70)
+2. table=49, priority=11800,conj_id=2,ip actions=load:0x2->NXM_NX_REG6[],resubmit(,70)
+3. table=49, priority=11800,ip,nw_src=10.10.1.6 actions=conjunction(2,1/3)
+4. table=49, priority=11800,ip,nw_dst=10.10.1.8 actions=conjunction(2,2/3)
+5. table=49, priority=11800,udp,tp_dst=53 actions=conjunction(2,3/3)
+6. table=49, priority=0 actions=resubmit(,50)
 ```
 
-Similar to K8s NetworkPolicy implementation, CnpEgressRuleTable also relies on
+Similar to K8s NetworkPolicy implementation, CnpEgressRuleTables also rely on
 the OVS built-in `conjunction` action to implement policies efficiently.
 The above example flows read as follow: if the source IP address is in set
 {10.10.1.6}, and the destination IP address is in the set {10.10.1.8}, and the
 destination TCP port is in the set {53}, then use the `conjunction` action with
-id 2, which goes to [L3ForwardingTable]. Otherwise, go to [EgressRuleTable].
+id 2, which goes to [L3ForwardingTable]. Otherwise, go to the table corresponding
+to the next tier in precedence, or in case of the Application tier (last in
```

Comment:
s/last in precedence/lowest precedence?

(same below for ingress)

Reply:
done